src/main/java/net/minecraftforge/client/event/RenderItemOverlayEvent.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2018.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.client.event;
+
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This event is called whenever {@link net.minecraft.item.Item#shouldRenderAdditionalOverlay(ItemStack)} returns true
```

Comment:
This, with the docs for `Item#shouldRenderAdditionalOverlay(ItemStack)` doesn't explain really when this is called.

Reply:
That would probably go under `Item#shouldRenderAdditionalOverlay(ItemStack) `, but I'm basing my javadocs on `Item#showDurabilityBar(ItemStack)` as it is called in the same method. `Item#showDurabilityBar(ItemStack)` doesn't specify *when* either. Also, it should be easy to look up where that method is being called in your IDE.